src/DynamoCoreWpf/Views/Gallery/GalleryView.xaml

```diff
@@ -101,15 +101,31 @@
             />
 
         <!-- Close button -->
-        <dynui:ImageButton 
-            Width="86" Height="21"
-            Grid.Column="1" Grid.Row="0"
+        <TextBlock 
```

Comment:
Two different way to solve it .one is to creat a new image (delete
term'close')and upload it, then we can use imagebutton. Another one is to
change it to text, which can keep term 'close'.
On Jul 6, 2015 3:35 PM, "Yu Ke" <notifications@github.com> wrote:

> In src/DynamoCoreWpf/Views/Gallery/GalleryView.xaml
> <https://github.com/DynamoDS/Dynamo/pull/4841#discussion_r33911227>:
>
> > @@ -101,15 +101,31 @@
> >              />
> >
> >          <!-- Close button -->
> > -        <dynui:ImageButton
> > -            Width="86" Height="21"
> > -            Grid.Column="1" Grid.Row="0"
> > +        <TextBlock
>
> Why it is a TextBox instead of a Button?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DynamoDS/Dynamo/pull/4841/files#r33911227>.
>